charts/vela-core/templates/defwithtemplate/import-grafana-dashboard.yaml

```diff
@@ -1,41 +0,0 @@
-# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
-# Definition source cue file: vela-templates/definitions/internal/import-grafana-dashboard.cue
-apiVersion: core.oam.dev/v1beta1
-kind: TraitDefinition
-metadata:
-  annotations:
-    definition.oam.dev/description: Import dashboards to Grafana
-  labels:
-    custom.definition.oam.dev/ui-hidden: "true"
-  name: import-grafana-dashboard
```

Comment:
why this can be deleted? \cc @Somefive

Reply:
This is the legacy trait used in observability addon in v1.1. Not in-use in current observability system.

Reply:
The current in-use trait for observability are integrated in addons.